components/sup/src/main.rs

```diff
@@ -132,6 +140,13 @@ fn sub_bash(m: &ArgMatches) -> Result<()> {
     command::shell::bash()
 }
 
+fn sub_config(m: &ArgMatches) -> Result<()> {
+    let ident = try!(PackageIdent::from_str(m.value_of("PKG_IDENT").unwrap()));
+
+    try!(common::command::package::config::start(&ident, Path::new("/")));
```

Comment:
Since you're using `AsRef<Path>` on `config::start()` you can actually just pass `"/"` instead of `Path::new("/")` 😄